### ISSUE TITLE
[Filebeat] Entity Analytics Input GA

### ISSUE THREAD

**Issue by jamiehynds** [elastic/beats#41492]
We now have several Entity Analytics integrations using the [EA input](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-entity-analytics.html) under the hood. 

We need to ensure the input is GA'd & fully supported and not left in Tech Preview for a long period of time. 

This issue will track any blockers (if any) to GA'ing the input and ensure we GA asap.

**Comment by narph** [comment]
@chemamartinez , @jamiehynds should we look into the input being supported by agentless?

**Comment by jamiehynds** [comment]
Definitely. It's not an optimal flow today where we tell people that can ingest events from Okta/EntraID etc agentlessly, but if they want to ingest user/group/device data from those same sources they have to install an agent.

**Comment by chemamartinez** [comment]
@andrewkroh defines in [this issue](https://github.com/elastic/security-integrations/issues/515) the required effort to support the Entity Analytics input in Agentless, so I'd like to discuss if this is a requirement for going GA with this input or can be handled separately.

Apart from that, the only known issue for the EA input in this repository is https://github.com/elastic/beats/issues/36447, doesn't seem critical but can be investigated.

I will also evaluate what is the current status of the input in terms of test coverage and usage.

Btw, adding some labels for most important inputs would be helpful in this repository to quickly identify issues and PRs.
